mcp/src/test/java/io/modelcontextprotocol/server/AbstractMcpClientServerIntegrationTests.java

```diff
@@ -1531,4 +1599,9 @@ private double evaluateExpression(String expression) {
 		};
 	}
 
+	protected static McpTransportContextExtractor<HttpServletRequest> TEST_CONTEXT_EXTRACTOR = (r, tc) -> {
+		tc.put("important", "value");
+		return tc;
+	};
```

Comment:
We could have a copy of this in both `HttpServletSseIntegrationTests` and `HttpServletStreamableIntegrationTests` to avoid a reference to a servlet type in the base class.

BTW, I thought this abstract test can be used outside of the servlet context, but it seems the variant in mcp-test module is not really a copy.. @tzolov do we have a strategy for aligning these? There's no comment like we have in other files of this sort (`// KEEP IN SYNC with the class in mcp-test module`)...